packages/wazuh_testing/wazuh_testing/fim.py

```diff
@@ -372,9 +436,12 @@ def callback_detect_end_scan(line):
 
 
 def callback_detect_event(line):
-    match = re.match(r'.*Sending event: (.+)$', line)
+    msg = r'.*Sending message to server: (.+)' if sys.platform == 'win32' else r'.*Sending event: (.+)$'
+    match = re.match(msg, line)
     if match:
-        if json.loads(match.group(1))['type'] == 'event':
+        if '{"type":"event"' not in line:
```

Comment:
First parse the json object and then check for 'type' key.

Reply:
Done. Now using try/except.